### PR TITLE
Floating Form

### PR DESCRIPTION
**Pull request by shatee**
localhost_9999

### CODE REVIEW

File: src/plugin/ClickableTaskLabelPlugin.js [hunk 1]
```diff
@@ -98,11 +99,8 @@ export class ClickableTaskLabelPlugin extends PluginInterface {
       assignees: this.roadmappy.roadmap.getAssignees().map(a => a.toAssoc()),
     });
 
-    // display in right down of the task
-    const taskLabelRect = labelNode.getBoundingClientRect();
-    const canvasRect = this.roadmappy.canvas.element.node().getBoundingClientRect();
-    this.form.style.left = `${taskLabelRect.x + taskLabelRect.width - canvasRect.x}px`;
-    this.form.style.top = `${taskLabelRect.y + taskLabelRect.height - canvasRect.y}px`;
+    this.form.style.left = `${d3.event.offsetX + 16}px`;
```
Comment (atago0129) [inline src/plugin/ClickableTaskLabelPlugin.js]: `16` is a magic number!!

File: src/plugin/ClickableTaskLabelPlugin.js [hunk 2]
```diff
@@ -97,6 +98,10 @@ export class ClickableTaskLabelPlugin extends PluginInterface {
       stories: this.roadmappy.roadmap.getStories().map(s => s.toAssoc()),
       assignees: this.roadmappy.roadmap.getAssignees().map(a => a.toAssoc()),
     });
+
+    this.form.style.left = `${d3.event.offsetX + 16}px`;
```
Comment (atago0129) [inline src/plugin/ClickableTaskLabelPlugin.js]: `16` is a magic number!!

File: src/plugin/ClickableTaskLabelPlugin.js [hunk 3]
```diff
@@ -25,13 +26,13 @@ export class ClickableTaskLabelPlugin extends PluginInterface {
...
-    this._initializeForm();
+    this._initializeForm(labelNode);
   };
 
   /**
-   * @param {RoadmapTask} task
+   * @param {Selection} labelNode
    */
-  _initializeForm() {
+  _initializeForm(labelNode) {
```
Comment (atago0129) [inline src/plugin/ClickableTaskLabelPlugin.js]: `labelNode` is an unused variable.

File: src/plugin/ClickableTaskLabelPlugin.js (hunk 2 above)
Comment (atago0129) [inline src/plugin/ClickableTaskLabelPlugin.js]: I think should to route mouse position without `d3.event.offsetX`
For example,
```
.on('click', (d) => this.emit('click:task-label', d, d3.event.target, d3.mouse(this.svg.node())[0], d3.mouse(this.svg.node())[1]))
```
in `RoadmapCanvas.js`

**Comment by shatee** [comment]
Following comment reflected 😃

**Pull request merged**